USFMToolsSharp/Models/Markers/Marker.cs

```diff
@@ -97,5 +97,15 @@ public List<Marker> GetHierarchyToMarker(Marker target)
 
             return output;
         }
+
+        public Marker GetLastChild()

```

Comment:
I think this method is misnamed.  I would expect `GetLastChild()` to return the last child (i.e. direct descendent) of this entity.  But it looks like this method returns the deepest descendent -- could be a child, grandchild, great-granchild, etc.  I think this method should be called something like `GetLastLeaf()` or `GetLastDescendent()`.